src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -179,6 +186,14 @@ def load_command_table(self, _):
         g.custom_command('delete', 'aks_snapshot_delete',
                          supports_no_wait=True)
 
-    # AKS trusted access roles commands
+    # AKS trusted access role commands
     with self.command_group('aks trustedaccess role', trustedaccess_role_sdk, client_factory=cf_trustedaccess_role) as g:
         g.custom_command('list', 'aks_trustedaccess_role_list')
+
+    # AKS trusted access rolebinding commands
+    with self.command_group('aks trustedaccess rolebinding', trustedaccess_role_binding_sdk, client_factory=cf_trustedaccess_role_binding) as g:
+        g.custom_command('list', 'aks_trustedaccess_role_binding_list')
+        g.custom_command('get', 'aks_trustedaccess_role_binding_get')
```

Comment:
Usually, the command corresponding to a get request is named as `show` in CLI. Please also update help and remove implementation from `custom.py`.
```suggestion
        g.show_command('show', 'get')
```